e2e/cypress/integration/messaging/tooltip_visual_verification_spec.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+// Username of a test user that you want to start a DM with
+let username = '';
+
+// Function to hover over individual items
+function hoverOverItem(postId, iconName, tooltipName, tooltipText, location = 'CENTER') {
+    // # Get the latest post and hover over it
+    cy.get(`#post_${postId}`).trigger('mouseover');
+
+    // # Hover on the icon that you want to test
+    cy.get(`#${location}_${iconName}_${postId}`).trigger('mouseover', {
+        force: true,
+    });
+
+    // # Check that after hovering on the icon you would like to test, the relevant tooltip appears
+    cy.get(`${tooltipName}`).find('span').invoke('text').then((text) => {
```

Comment:
Can be shortened to  `cy.get('${toolipName}').find('span').should('have.text', tooltipText)`

Reply:
Done